dmeta/params.py

```diff
@@ -36,3 +36,8 @@
     "pptx",
     "xlsx"
 ]
+NOT_IMPLEMENTED_ERROR = "Removing the meta data of this format is not yet supported by DMeta."
+FILE_FORMAT_DOES_NOT_EXIST_ERROR = "Given file name doesn't have a specific format at the end."
```

Comment:
`Failed to detect file format. Make sure the file has proper extension (.docx, .pptx, etc.).`